### DISCUSSION TITLE
Pulling (Requests) our weight 🏋️ [Pull Requests Mini Check-in]

### DISCUSSION THREAD

**Discussion by queenofcorgis** [community/community#62082] [Pull Requests]
We’ve been hard at work iterating and improving on pull requests, almost as hard as you’ve been working on giving your feedback, thank you so much for lifting the bar even higher for us. 

⚡**Pull Requests Praise**: Thanks to these fine folks for keeping our Community awesome ✔️
* @amarzot: For sharing how a user could free up valuable screen space on the sidebar. Check it out [here](https://github.com/orgs/community/discussions/61320). 
* @rtalexk: Posed an interesting question about search syntax in [their discussion](https://github.com/orgs/community/discussions/60164). Go check it out! 

   🏅 Want to nominate a community member for a feature above? Just tag me (@queenofcorgis) where you’ve seen great community behavior.  🏅 

ℹ️ **What’s new:** The latest Pull Requests updates

* **Pull request merge queue is now generally available!**: Merge queue helps increase velocity in software delivery by automating pull request merges into your busiest branches. [View the changelog](https://github.blog/changelog/2023-07-12-pull-request-merge-queue-is-now-generally-available/). 🎉
* **Rebase commits now created using the `merge-ort` strategy**: Rebase commits are speedier than ever which drops rebases that fail due timeout. Get all the details through this [link](https://github.blog/changelog/2023-06-28-rebase-commits-now-created-using-the-merge-ort-strategy/). 

🆒 **Cool stuff™️ happening around GitHub and the Community**

* **Metrics for issues, pull requests, and discussions**: With the new Issue Metrics GitHub Action, you can now track and monitor important metrics related to issues, pull requests, and discussions, such as time to first response, time to close, and more! [Learn more with our blog post](https://github.blog/2023-07-19-metrics-for-issues-pull-requests-and-discussions/). 
* **Demonstrating end-to-end traceability with pull requests**: Reduce developer and auditor friction involved in demonstrating compliance and maintaining end-to-end traceability by focusing your efforts around the pull request. [Read more](https://github.blog/2023-07-11-demonstrating-end-to-end-traceability-with-pull-requests/) with @pholleran on our blog. 

That’s it for this set 👟. See you next month!

**Comment by ghost** [comment]
Metrics for issues, pull requests, and discussions: With the new Issue Metrics GitHub Action, you can now track and monitor important metrics related to issues, pull requests, and discussions, such as time to first response, time to close, and more!

**Reply by jandsonrj** [reply]
With the introduction of the new GitHub action called "Issue Metrics GitHub Action", you now have the ability to effectively track and monitor key metrics associated with issues, pull requests, and discussions. These metrics include, but are not limited to, the time taken to get the first response, as well as the length of time until the issue or pull request is closed. With this functionality, you gain valuable insight into your project workflow, enabling deeper analysis and informed decision making. This results in a more efficient and transparent development process, promoting collaboration among team members and improving the overall quality of work.

**Reply by Harshsanas** [reply]
Time to First Response: This metric measures how long it takes for the first response to be made on an issue or pull request after it's been opened. A quick response time indicates an active and engaged development community.

Time to Close: This metric calculates the time it takes to close an issue or merge a pull request from the moment it was opened. A shorter time to close generally signifies an efficient development process.

Discussion Engagement: This metric could involve tracking the number of comments, participants, or reactions on an issue or pull request. It provides insight into the level of collaboration and discussion around a particular topic.

Issue and Pull Request Throughput: This metric looks at the rate at which issues and pull requests are opened and closed or merged. It helps you understand the flow of work within the repository.

Backlog Size: Monitoring the number of open issues and pull requests gives you an idea of the current workload and helps identify potential bottlenecks in your development process.

Average Time in Each Stage: For pull requests, you could track the average time spent in different stages, such as review, testing, and approval. This helps pinpoint areas that might need improvement.

Churn Rate: This metric shows the rate at which issues or pull requests are opened and then subsequently closed without being addressed. A high churn rate could indicate that issues are being opened prematurely or without enough information.

**Comment by mdtanvirahamedshanto** [comment]
yes

**Comment by queenofcorgis** [comment]
Take a [look at our latest check-in](https://github.com/orgs/community/discussions/64874) to see category superstars and a feedback opportunity!
